api/turing/api/deployment_controller.go

```diff
@@ -161,7 +161,7 @@ func (c RouterDeploymentController) deployRouterVersion(
 
 	if routerVersion.LogConfig.ResultLoggerType == models.BigQueryLogger {
 		routerServiceAccountKey, err = c.MLPService.GetSecret(
-			models.ID(project.Id),
+			models.ID(project.ID),
```

Comment:
This change is made due to an update in the `gojek/mlp` version which features new attribute names for the `Project` struct.